orders/views.py

```diff
@@ -27,4 +27,24 @@ def get(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
         
         except User.DoesNotExist:
-            return JsonResponse({'message':'INVALID_USER'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'INVALID_USER'}, status=400)
+
+    @login_decorator
+    def delete(self, request, cart_item):
+        try:
+            signed_user = request.user
+
+            if cart_item == 0:
+                items = CartItem.objects.filter(user=signed_user)
+                for item in items:
+                    item.delete()
+                return JsonResponse({'message':'DELETE_SUCCESS'}, status=204)
+
+            if not CartItem.objects.filter(pk=cart_item, user=signed_user).exists():
+                return JsonResponse({'message':'INVALID_ITEM'}, status=400)
```

Comment:
```suggestion
            if not CartItem.objects.filter(pk=cart_item, user=signed_user).exists():
                return JsonResponse({'message':'NOT_FOUND'}, status=404)
```
restful하게 바라봤을 때 결국 id가 없다는 것은 해당 url이 존재하지 않은 것으로 판단 되므로 404가 조금 더 어울릴 것 같습니다.

Reply:
넵 감사합니다! 404로 수정 완료 했습니다!